diuf/sudoku/Cell.java

```diff
@@ -92,6 +92,21 @@ public CellSet getVisibleCells() {
     	return Grid.visibleCellsSet[index];
     }
 
+   public boolean canSeeCell(Cell other) {
+	   return Grid.visibleCellsSet[index].contains(other);
+   }
+   

```

Comment:
canSeeCell() unless in blind mode?
Isn't seesCell() more descriptive?

Reply:
BTW, seesCell() works too!!